rootfs/scheduler/__init__.py

```diff
@@ -1004,12 +1004,7 @@ def _scale_rc(self, namespace, name, desired):
             'version': rc['spec']['selector']['version']
         }
 
-        # Are there any pods running (and verified as ready) available?
-        pods = self._get_pods(namespace, labels=labels).json()['items']
-        current = 0
-        for pod in pods:
-            if self._pod_ready(pod):
-                current += 1
+        current = int(rc['spec']['replicas'])
```

Comment:
when i do deis scale web/cmd=<some number> i meant that the number is the desired number of replicas i want eventually which is spec/replicas.
whereas status/replicas is observed number of replicas at that time....which can be different during the scheduling of the pod but will eventually be same as spec/replicas